openapi/generator_test.go

```diff
@@ -260,6 +295,12 @@ func TestNewSchemaFromStructFieldErrors(t *testing.T) {
 	assert.Len(t, g.Errors(), 4)
 	assert.NotEmpty(t, g.Errors()[2].Error())
 	assert.NotEmpty(t, g.Errors()[3].Error())
+
+	// Field D has example value that cannot be parsed to bool.
+	sor = g.newSchemaFromStructField(typ.Field(3), false, "D", typ)
+	assert.NotNil(t, sor)
+	assert.Len(t, g.Errors(), 5)
+	assert.Contains(t, g.Errors()[4].Error(), "could not parse the example value \"not-a-bool-value\" of field \"D\"")
```

Comment:
I wasn't thinking of that kind of test, perhaps I wasn't clear enough.
What I had in mind was a type cast f the error to `fizz.FieldError`, then check if the fields `Name`, `Typ` values match what we expect to see. The format of the error itself isn't that relevant.

Reply:
However, we need the previous `assert` that checked if the return of `Error()` isn't empty.

Reply:
Ohhhh ... I think I get now what you meant. Just pushed, please check 🙏